docs/sources/project/find-an-issue.md

```diff
@@ -195,12 +195,12 @@ To sync your repository:
           (use "git push" to publish your local commits)
         nothing to commit, working directory clean
 
-    Your local repository now has any changes from the `upstream` remote.  You
+    Your local repository now has many changes from the `upstream` remote.  You
```

Comment:
I think "any" is correct here; it could well be (although unlikely :smile:) that there were no changes upstream